chatkit-core/config/baseline.xml

```diff
@@ -2,41 +2,26 @@
 <SmellBaseline>
   <Blacklist></Blacklist>
   <Whitelist>
-    <ID>LongMethod:RoomStore.kt$RoomStore$applyUserSubscriptionEvent</ID>
-    <ID>LongParameterList:CurrentUser.kt$CurrentUser$( id: String? = null, name: String, pushNotificationTitleOverride: String? = null, isPrivate: Boolean = false, customData: CustomData? = null, userIds: List&lt;String&gt; = emptyList(), callback: (Result&lt;Room, Error&gt;) -&gt; Unit )</ID>
-    <ID>LongParameterList:CurrentUser.kt$CurrentUser$( room: Room, name: String, pushNotificationTitleOverride: RoomPushNotificationTitle? = null, isPrivate: Boolean? = null, customData: CustomData? = null, callback: (Result&lt;Unit, Error&gt;) -&gt; Unit )</ID>
-    <ID>LongParameterList:CurrentUser.kt$CurrentUser$( roomId: String, name: String, pushNotificationTitleOverride: RoomPushNotificationTitle? = null, isPrivate: Boolean? = null, customData: CustomData? = null, callback: (Result&lt;Unit, Error&gt;) -&gt; Unit )</ID>
+    <ID>EmptyFunctionBlock:CursorService.kt$CursorService${}</ID>
+    <ID>EmptyFunctionBlock:MessageService.kt$MessageService${}</ID>
+    <ID>EmptyFunctionBlock:RoomService.kt$RoomService${}</ID>
+    <ID>EmptyFunctionBlock:UserService.kt$UserService${}</ID>
```

Comment:
bits for the removed services can be removed from the whitelist

Reply:
I ran the detektBaseline gradle task and it removed a bunch of other whitelisted things that are no longer issues :)